flight_phase_files/HALO_RF15_20200218_info.yaml

```
@@ -0,0 +1,76 @@
+name: RF15
+mission: EUREC4A
+platform: HALO
+flight_id: HALO-0218
+contacts:
+- name: Marc Prange
+  email: marc.prange@uni-hamburg.de
+  tags:
+  - lc
```

I think Marc is also to be tagged as dc. At least, that's what I did for my flights.
And I think that's the only suggested change I could find. :)
```suggestion
  tags:
  - lc
  - dc
```